stage1/usr_from_kvm/files/sshd_config

```diff
@@ -0,0 +1,36 @@
+Port 22
```

Comment:
This doesn't matter, we are using inetd mode of ssdh running (-i parameter) for "old style" socket activation.
I'll try to remove this line (should be working ok), but at this moment I don't have time to test this, so this could be done after merge in separate small issue.